ACE/ace/config-sunos5.10.h

```diff
@@ -59,4 +59,9 @@
 
 #define ACE_HAS_SOLARIS_ATOMIC_LIB
 
+// Solaris Studio 12.4 implements symbol lookup correctly.
+#if defined (__SUNPRO_CC) && (__SUNPRO_CC >= 0x5130)
+#define ACE_ANY_OPS_USE_NAMESPACE
+#endif
```

Comment:
Adding the symbol (conditionally) in sunos5.10 should be fine, because the latest compilers for which it matters, 12.4 and 12.5, don't even offer installation on any older sunos than 5.10.

config-suncc-common seems to be only included by config-linux.